index.bs

```diff
@@ -2190,8 +2190,11 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is present with the value
     {{CredentialMediationRequirement/conditional}}:
 
-    1. If <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> is not [=list/empty=],
-        return a {{DOMException}} whose name is "{{NotSupportedError}}", and terminate this algorithm.
+    1. Let |credentialIdFilter| be <code>|options|.{{CredentialRequestOptions/publicKey}}.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code>.
+
+    1. Set <code>|options|.{{CredentialRequestOptions/publicKey}}.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> to [=list/empty=].
+
+        Note: This prevents [=non-discoverable credentials=] from being used during {{CredentialMediationRequirement/conditional}} requests.
```

Comment:
I recall that we said at TPAC and in https://github.com/w3c/webauthn/issues/1793#issuecomment-1244413340 that conditional UI wouldn't support non-discoverable credentials, but I don't recall the motivation. Why add that restriction?

Reply:
Essentially the same reasoning as in the [explainer](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI#empty-allowcredentials): non discoverable credentials don't usually store `PublicKeyCredentialUserEntity` information, so we wouldn't be able to display them on the autofill surface. Silent discovery would not be available for non discoverable credentials on platform authenticators.

(I need to go and update the explainer with this new filtering)

Reply:
Ah, I see. And I suppose that restricting to only discoverable keys prevents any confusion around why some particular authenticator's non-discoverable keys work with conditional UI (because it happens to store those values anyway) while another's doesn't. Fair enough, then. Thanks!